src/patternfly/components/Table/examples/table-expandable-example.hbs

```diff
@@ -26,7 +26,8 @@
         <input type="checkbox" name="checkrow1" aria-labelledby="{{concat table--id '-node1'}}" aria-label="Details">
       {{/table-td}}
       {{#> table-th table-th--data-label="Repository Name"}}
-        <div id="{{table--id}}-node1">Node 1</div>
+        <p id="{{table--id}}-node1">Node 1</p>
```

Comment:
We don't want to assume that consumers to use a `<p>` to make this work. That's what we thought that wrapping anything they put inside of the `<div>` will work.

Reply:
OK, I see. How about this as an alternative to wrapping the contents:
If you give the `:before` a sufficiently large `grid-row-end` then it will take up whatever grid cells get created on the right half of the grid. I thought there was a name for the last line, but I can’t find that there is unless you define a grid template, which you couldn’t do here, you need an implicit grid to handle whatever they put in the TD. But if you put a very large number, it does not cause any problem because there is no `grid-row-gap`.

Reply:
Ah awesome this works! Thanks @srambach

Reply:
Actually.... @srambach , @mcoker pointed out to me that this method won't fully work because when we have  `Content <span> Content <span/>` The content in the span will still drop to a second line.. However I may not have explained that in the comments, so thank you for taking the time to do this!